src/Azure.Functions.ArtifactAssembler/ArtifactAssembler.cs

```diff
@@ -181,11 +207,125 @@ private async Task CreateVisualStudioCoreToolsAsync()
             });
 
             await Task.WhenAll(packTasks);
+            Console.WriteLine($"Finished assembling Visual Studio Core Tools artifacts");
+        }
+
+        private async Task CreateCliCoreToolsAsync()
+        {
+            Console.WriteLine($"Starting to assemble CLI Core Tools artifacts");
+
+            // Create a directory to store the assembled artifacts.
+            var cliCoreToolsTargetArtifactDir = Path.Combine(_stagingDirectory, CliOutputArtifactDirectoryName);
+            Directory.CreateDirectory(cliCoreToolsTargetArtifactDir);
+            string outOfProcVersion = string.Empty;
```

Comment:
nit: doesn't matter either way, but can be simplified like this:

```csharp
string outOfProcVersion = string.Empty, 
       inProcVersion = string.Empty, 
       outOfProcArtifactDirPath = string.Empty, 
       inProc8ArtifactDirPath = string.Empty;
```

Reply:
done!